src/main/java/models/member/Member.java

```diff
@@ -32,9 +32,7 @@ public boolean equals(Object obj) {
             return false;
         } else {
             Member other = (Member) obj;
-            return this.name.equals(other.name)
-                && this.phone.equals(other.phone)
-                && this.email.equals(other.email);
+            return this.name.equals(other.name);
         }
```

Comment:
What's the reason for this change?

Reply:
Previously the hashing function made it such that members with the same name could be added, which could cause confusion. Furthermore it could result in problematic distribution of credits (as mentioned in issue #282 To prevent this, we ensure that members must have different names.

Reply:
yeap, seems like a logical explanation